script/Arcueid/Day 7/07_00_ARC07_3_1_3.txt

```diff
@@ -479,7 +479,7 @@ The flames he spread just so he could bear that cold.
 [sha:e394f35e6b609be888851eec7680c8fd8e6d31c2]{
 -- Page 606, Offset 18098.
 -- これはもう、『人のカタチをしたもの』が持って許されるものではない。
-That's not the sort of power that anything of human origin should be able to obtain.
+That sort of power should be beyond the reaches of anything of human origin.
```

Comment:
Idk why I can't get rid of the extra space before "Sort" here.

Reply:
The highlighted space just means that the space was added in place of the `'s not the ` that was removed, there's only one space there

Reply:
Mild pref for original

Reply:
I'm in agreement that "That's not the sort of power" should be kept